src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SingleLineJavadocCheck.java

```diff
@@ -61,6 +61,20 @@
  * <pre>
  * &lt;module name=&quot;SingleLineJavadoc&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * &#47;** &#64;return 42 *&#47; // violation, javadoc should be multiline
```

Comment:
please use the same javadoc/java for all examples blocks.
only configuration is changing, so only trailing comments (ok, violation) are changing also.
each example should have violation and ok code.

Reply:
no done. Java examples are still different.

@the-first-avenger , please follow this review point.
As you fix it please reply as "done".

Reply:
it is better to use only one new property in comparison to default config it help user see effect of one property to behavior.

Reply:
@romani Can you please point out the difference that I should fix ? I know there are some differences in the javadoc but those are necessary to show different options. For example, only the `violateExecutionOnNonTightHtml` cannot be shown using a `@see` tag.

Reply:
![image](https://user-images.githubusercontent.com/812984/108082388-e3550480-7026-11eb-86e2-64d9cb38e427.png)

please merge all example blocks in one, and use it for all configs.
Example: https://checkstyle.org/config_annotation.html#AnnotationLocation_Examples

Reply:
please make examples the same and extend them to be responsive (changes in tailing comments) on each config change.

Reply:
@romani Apologies for the delay was busy IRL. I have updated the PR according to your comments.